ajouter.php

```diff
@@ -0,0 +1,24 @@
+<?php require 'header.php'; ?>
```

Comment:
Petite préférence pour l'utilisation de `require_once`
- **include** ajoute le code demandé sans erreur si la page n'existe pas
- **include_once** est comme include mais ajoute le code une seule fois
- **require** ajoute le code demandé et lève une erreur si la page n'existe pas
- **require_once** est comme require mais ajoute le code une seule fois

Dans ton cas tu as besoin d'être sur d'insérer l'header => require 
Mais tu veux aussi être sûr qu'il ne sera ajouté qu'une seule fois dans la page (sinon erreur HTML) => require_once

Reply:
c'est noté, c'est fait